src/admin-views/settings/sidebars/default-sidebar.php

```diff
@@ -45,81 +51,104 @@
 		)
 	)
 );
-$hero_section->set_title( new Heading( __( 'Finding and extending your tickets', 'event-tickets' ), 2, new Classes( 'tec-settings-form__sidebar-header' ) ) );
 
-$hero_section->add_section(
-	( new Settings_Section() )
-		->add_elements(
-			[
-				( new Paragraph() )->add_child(
-					new Plain_Text( __( 'Looking for additional functionality including recurring events, custom meta, community events, ticket sales, and more?', 'event-tickets' ) )
-				),
-				new Link(
-					'https://evnt.is/products',
-					__( 'Check out the available add-ons.', 'event-tickets' ),
-					null,
-					$external_attributes
-				),
-			]
-		)
+// Set Sidebar Title.
+$hero_section->set_title(
```

Comment:
This is a fluent interface, you _can_ just chain these. i.e.
`$hero_section->set_header_image( ... )->set_title( ... )->etc()`

Not saying you _have_ to.